### PR TITLE
Fix starting transaction coordinator even if the local transactions are used

### PR DESCRIPTION
**Pull request by lasinicl**

### CODE REVIEW

**Review by gimantha — APPROVED** [review]

File: transaction-ballerina/internal.bal [hunk 1]
```diff
@@ -39,6 +40,16 @@ function startTransaction(string transactionBlockId, lang_trx:Info? prevAttempt
      }
 }
 
+function startTransactionCoordinator() {
+    http:Listener coordinatorListener = new http:Listener(coordinatorPort, { host: coordinatorHost });
+    //attach initiatorService to listener
+    error? attachInitiatorService = coordinatorListener.attach(initiatorService, "/balcoordinator/initiator");
+    // attach participant2pcService to listener
+    error? attachParticipant2pcService = coordinatorListener.attach(participant2pcService, "/balcoordinator/participant/'2pc");
```
Comment (gimantha) [inline transaction-ballerina/internal.bal:48]: path has to be changed from `2pc to 2pc

File: transaction-ballerina/tests/transactional_services.bal [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import ballerina/http;
```
Comment (gimantha) [inline transaction-ballerina/tests/transactional_services.bal:1]: Lets add copyright notes

File: transaction-ballerina/internal.bal (hunk 1 above)
Comment (lasinicl) [reply transaction-ballerina/internal.bal:48]: This was fixed in https://github.com/ballerina-platform/module-ballerinai-transaction/pull/33

File: transaction-ballerina/tests/transactional_services.bal (hunk 2 above)
Comment (lasinicl) [reply transaction-ballerina/tests/transactional_services.bal:1]: Added in https://github.com/ballerina-platform/module-ballerinai-transaction/pull/40/commits/ba3852ff7b6fab739a754330297a06682f4adab4

**Pull request merged**